src/main/java/org/folio/oaipmh/Constants.java

```diff
@@ -25,6 +23,7 @@ private Constants() {
 
   public static final String REPOSITORY_BASE_URL = "repository.baseURL";
   public static final String REPOSITORY_MAX_RECORDS_PER_RESPONSE = "repository.maxRecordsPerResponse";
+  public static final String REPOSITORY_HTTP_REQUEST_RETRY_ATTEMPTS = "repository.httpRequestRetryAttempts";
```

Comment:
`httpRequestRetryAttempts` is a little misleading - The module makes several HTTP requests.  This configuration only applies to the call to SRS.

Reply:
Okay, it makes sense. I added the "srs" prefix to the property name.